utils/sql/git/required/2022_08_01_splash_faction.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE `splash_faction` (
+	`id` INT(10) NOT NULL,
+	`splash_id1` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod1` FLOAT NOT NULL DEFAULT 0,
+	`splash_id2` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod2` FLOAT NOT NULL DEFAULT 0,
+	`splash_id3` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod3` FLOAT NOT NULL DEFAULT 0,
+	`splash_id4` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod4` FLOAT NOT NULL DEFAULT 0,
+	`splash_id5` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod5` FLOAT NOT NULL DEFAULT 0,
+	`splash_id6` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod6` FLOAT NOT NULL DEFAULT 0,
+	`splash_id7` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod7` FLOAT NOT NULL DEFAULT 0,
+	`splash_id8` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod8` FLOAT NOT NULL DEFAULT 0,
+	`splash_id9` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod9` FLOAT NOT NULL DEFAULT 0,
+	`splash_id10` INT(10) NOT NULL DEFAULT 0,
+	`splash_mod10` FLOAT NOT NULL DEFAULT 0,
+	PRIMARY KEY(`id`)
+);
```

Comment:
Could probably make this into a table that just links a primary faction with a sub faction id and a multiplier,
something like:
`primary key id autoincrement, primary_faction_id, sub_faction_id, mod/multiplier`

Not sure if that would make things more complicated or not, but wouldn't need to worry about reaching any limit (though 10 seems high already)

Reply:
I would rather keep it simple. This isn't about giving us the ability to do anything more than what live does. And I would rather increase the limit than making it arbitrary if we come across a case that live needs.

Reply:
curious why these wouldn't just be additional columns on the faction table?

Reply:
I didn't really want to touch the current stuff and risk breaking anything. Plus joins are involved in loading faction stuff currently and I didn't wanna deal with that :D

Reply:
Okay, looked into how `npc_faction` and `npc_faction_entries` works ... it doesn't really fit in there :P

Reply:
If it were in an existing table I would expect it in faction_base_data or faction_list tbh

Reply:
faction_base_data could probably be expanded for this.

Reply:
> faction_base_data could probably be expanded for this.

One argument against is that faction_base_data is a direct import from client files, or was anyway.